lib/ckb/address.rb

```diff
@@ -52,15 +52,15 @@ def generate_short_payload_hash160_address(hash160)
     # @return [String]
     def generate_full_payload_address(format_type, code_hash, args)
       format_type = Utils.to_hex(format_type)[2..-1].rjust(2, '0')
-      raise "Invalid format type" unless TYPES[1..-1].include?(format_type)
-      raise "Args should be an array" unless args.is_a?(Array)
+      raise InvalidFormatTypeError.new("Invalid format type") unless TYPES[1..-1].include?(format_type)
+      raise InvalidArgsTypeError.new("Args should be an array") unless args.is_a?(Array)
 
       payload = [format_type].pack("H*") + CKB::Utils.hex_to_bin(code_hash)
       args.each do |arg|
         arg_bytes = CKB::Utils.hex_to_bin(arg)
         arg_len = arg_bytes.bytesize
         if arg_len > 256 || arg_len == 0
-          raise "Arg size is too large"
+          raise InvalidArgSizeError.new("Arg size is too large")
```

Comment:
It would be a surprise if `arg_len == 0` and error message is `Arg size is too large`.

Reply:
thank. yes, I will fix this.

Reply:
after discussing with the Cipher, decided to allow empty arg.

see: https://github.com/CipherWang/ckb-address-demo/pull/1